python/taichi/profiler/__init__.py

```diff
@@ -0,0 +1,2 @@
+from .kernelprofiler import KernelProfiler  # import for docstring-gen
```

Comment:
nit: full import path

Reply:
Some functions are being refactored, I'll add docstings for the public methods ASAP.